app/uk/gov/hmrc/exports/repositories/MigrationsTransactionalOps.scala

```diff
@@ -31,7 +31,7 @@ import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class TransactionalOps @Inject() (
+class MigrationsTransactionalOps @Inject() (
```

Comment:
I would rename this to SubmissionTransactionalOps, or something like that. Migrations is too generic.